src/uwtools/cli.py

```diff
@@ -633,6 +633,7 @@ def _abort(msg: str) -> None:
     :param msg: The message to print.
     """
     print(msg, file=sys.stderr)
+    log.exception(msg)
```

Comment:
@WeirAE Is addressing this elsewhere, right? I think you probably just committed some debug code...

Reply:
Ugh. I committed and pushed some things I did not mean to!